### ISSUE TITLE
Excluding visited URLs from LinkHints

### ISSUE THREAD

**Issue by kklot** [gdh1995/vimium-c#1125]
**What command or commands**

LinkHints' command

**How should a feature do**

Option for hints to only appears on non-visited URL, such as instead of `f` pressing `Alt+F`. 

For my case, forums page can have 3 letters hints which is a bit longer to recognize and type.

**Comment by chjayakrishnajk** [comment]
You can try to increase characters used in link hints by going to vimium-c options and adding more characters to "Characters used for link hints"
https://www.calculatorsoup.com/calculators/discretemathematics/permutations.php
Use this website to guess how many characters you would need by seeing permutations
Eg: if you input 13 and 2, it will give 156 permutations, so it can create link hints of max 156 with only 2 letters, if you website has 200 links then slowly increase the input of n, and you'll find out that when you use 15 characters it can create 210 link hints

**Comment by gdh1995** [comment]
Yes a longer value for "Characters used for link hints" should help.

The "max number of marks" is `N * (N - 1)` if there're `N` characters in the option.

The idea to exclude visited URLs may hurt your work experience, because you'll notice some usual links are "almost never" hinted.

What's worse is, extensions can not learn "whether a link has been visited or not" from your browser easily.
